study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/StudyBuilder.java

```diff
@@ -203,6 +205,40 @@ private Auth0TenantDto getTenantOrInsert(Handle handle) {
         return dto;
     }
 
+    public void insertTenantPasswordPolicy(Handle handle, Auth0TenantDto tenantDto) {
+        if (cfg.hasPath("tenant.passwordPolicy")) {
+            JdbiStudyPasswordRequirements jdbiPasswordPolicy = handle.attach(JdbiStudyPasswordRequirements.class);
+            StudyPasswordRequirementsDto policyDto = jdbiPasswordPolicy.getById(tenantDto.getId()).orElse(null);
+            if (policyDto == null) {
```

Comment:
My be just return early to reduce nesting levels? I.e. `if (policyDto != null) { ... return }`